src/query/sql/src/planner/semantic/type_check.rs

```diff
@@ -931,6 +932,7 @@ impl<'a> TypeChecker<'a> {
                 args[1] = ConstantExpr {
                     value: common_expression::Literal::Int64(1),
                     data_type: Box::new(DataType::Number(NumberDataType::Int64)),
+                    key: None,
```

Comment:
Adding a `key` in `ConstantExpr` seems strange. How about adding a new expr type? like `AssignmentExpr`?

Reply:
I have tried adding a new expr, but it introduced many modifications. I think make it as a `ConstantExpr` would make things simpler.